package-lock.json

```diff
@@ -10692,9 +10692,9 @@
       }
     },
     "node_modules/@tiptap/core": {
-      "version": "2.1.11",
-      "resolved": "https://registry.npmjs.org/@tiptap/core/-/core-2.1.11.tgz",
-      "integrity": "sha512-1W2DdjpPwfphHgQ3Qm4s5wzCnEjiXm1TeZ+6/zBl89yKURXgv8Mw1JGdj/NcImQjtDcsNn97MscACK3GKbEJBA==",
+      "version": "2.1.12",
```

Comment:
@jattasNI know if we can configure it to merge instead of force push? Reason is it's doing all the intermediary builds but we lose the status checks for them in the commit list since the history is gone. Although if not could dig into the build logs to find it.

i.e. while reviewing this PR recent merges in main mean I can't check storybook till the build finishes or jumping though small hoops to find the old build

Reply:
I don't see any configuration for that mentioned in their [docs on updating and rebasing branches](https://docs.renovatebot.com/updating-rebasing). In fact it says:
>Renovate uses its own version of "rebasing", which is not the same as doing a git rebase with Git. Instead, Renovate reapplies all updates into a new commit based off of the head of the base branch.

Something we do have control over is how often it rebases, via the [rebaseWhen setting](https://docs.renovatebot.com/configuration-options/#rebasewhen). It currently rebases whenever its branch falls behind main because this repo requires branches to be up to date before merging. An alternative would be setting it to rebase only if there are conflicts or rebase never. Then we would have to manually update it before merging, by clicking the checkbox in the PR description. That doesn't sound too burdensome to me, and it would save churn on our agents. What do you think @rajsite?

Reply:
Cool, I can make this change.

Reply:
#1640

Reply:
Unresolving guess we can validate on this PR

Reply:
It's now showing the normal status we see on other PRs about "This branch is out-of-date with the base branch" so I think we successfully caused it to stop updating. I'll let you have the satisfaction of closing this comment thread, enabling automerge, and clicking the checkbox in the PR description.

I guess one thing to note is that we might want to prefer that checkbox vs the typical "Update branch" button since it keeps Renovate in control of the commit history. Letting GitHub push to this branch might cause Renovate to stop updating it, which would mean that it couldn't automatically resolve package-lock conflicts anymore. (At least until you click the checkbox, so may as well do that from the beginning)

Reply:
true!

Reply:
> I guess one thing to note is that we might want to prefer that checkbox vs the typical "Update branch" button since it keeps Renovate in control of the commit history

@jattasNI  I said true and resolved this thread but not sure the best place to capture that. Can we customize the PR description renovate makes to add that note in bold or something?

Reply:
Since it'll just be you and me merging these PRs for now, I'd vote for something simple like hoping we remember or updating the spot in CONTRIBUTING where we talk about Renovate.

Reply:
#1641